spec/models/spree/billing_integration_spec.rb

```diff
@@ -0,0 +1,26 @@
+describe 'Deprecated BillingIntegration' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.